<launch>
    
    <node name="kittibag" pkg="kittibag" type="kittibag" output="screen"> 
        <param name="dataset_folder" type="string" value="/home/cjd/dataset/KITTI_360/" />
        <param name="sequence_number" type="string" value="2013_05_28_drive_0002_sync" />
        <param name="to_bag" type="bool" value="true" />
        <param name="output_bag_file" type="string" value="/home/cjd/dataset/KITTI_360/bag/2013_05_28_drive_0002_sync.bag" /> <!-- replace with your output folder -->
        <param name="publish_delay" type="int" value="1" />
    </node>
</launch>
